modules/RTC/TraceablePeerConnection.js

```diff
@@ -205,13 +206,21 @@ export default function TraceablePeerConnection(
      */
     this.localSdpMunger = new LocalSdpMunger(this);
 
+    this.bandwidthLimiter = new BandwidthLimiter();
+
     /**
      * TracablePeerConnection uses RTC's eventEmitter
      * @type {EventEmitter}
      */
     this.eventEmitter = rtc.eventEmitter;
     this.rtxModifier = new RtxModifier();
 
+    /**
+     * Whether or not this endpoint has been selected
```

Comment:
do you utilise full line length of 80 in this file ?

Reply:
fixed